js/data/pokedex.js

```diff
@@ -10579,5 +10579,75 @@ var POKEDEX_XY = $.extend(true, {}, POKEDEX_BW, {
     },
     "w": 305.0,
     "ab": "Aura Break"
+  },
+  "Zygarde-Cell": {
+    "t1": "Dragon",
+    "t2": "Ground",
+    "bs": {
+      "hp": ,
```

Comment:
It's no big deal, so no need to fix it now, it's just for future reference. Invalid code just makes me vaguely uncomfortable. :P